packages/interactive_media_ads/android/src/main/kotlin/dev/flutter/packages/interactive_media_ads/AdsRequestProxyApi.kt

```diff
@@ -21,7 +21,7 @@ class AdsRequestProxyApi(override val pigeonRegistrar: ProxyApiRegistrar) :
      *
      * This must match the version in pubspec.yaml.
      */
-    const val pluginVersion = "0.2.3+4"
+    const val pluginVersion = "0.2.3+5"
```

Comment:
@bparrishMines is this how it's expected to work: whenever the version is updated this proxy API is bumped as well? Is there any other implication to doing this?

Reply:
Yes, this is what is expected with version bumps! This value is only used to append to AdRequests.